javascript-carousel/main.js

```diff
@@ -0,0 +1,81 @@
+var pictures = document.querySelectorAll('img');
+var buttons = document.querySelectorAll('.fa-circle');
+var buttonRow = document.querySelector('.row2');
+var arrowRow = document.querySelector('.row1');
+var currentNumber = 0;
+
+function buttonSwap(event) {
+  console.log('clic');
```

Comment:
Remove any console logs before submitting the exercise! Unless the instructions specifically ask for console logs